Lib/pkgutil.py

```diff
@@ -465,6 +465,10 @@ def get_loader(module_or_name):
     If the named module is not already imported, its containing package
     (if any) is imported, in order to establish the package __path__.
     """
+    warnings.warn("`pkgutil.get_loader` is deprecated since Python 3.12; "
+                  "this function is slated for removal in Python 3.14, "
+                  "use `importlib.util.find_spec` instead",
+                  DeprecationWarning)
```

Comment:
Better yet, use the built-in `warnings._deprecated` that is specifically intended for this purpose (standard library deprecations), which not only handles the `stacklevel` automatically, but also automatically warns if the removal is done missed before the first beta of the removal version, is more DRY/concise, and uses a standard, single-source-updatable and more easily parsable format.

```suggestion
    warnings._deprecated("pkgutil.get_loader",
                         f"{warnings._DEPRECATED_MSG}; "
                         "use importlib.util.find_spec instead",
                         remove=(3, 14))
```

I intend to update `warnings._deprecated` to add a `replace` arg, so this will be able to be just

```python
    warnings._deprecated(
        "pkgutil.get_loader", remove=(3, 14), replace="importlib.util.find_spec")
```